www/artists/delete.php

```diff
@@ -0,0 +1,82 @@
+<?
+use function Safe\session_unset;
+
+$artist = null;
+
+try{
+	if(Session::$User === null){
+		throw new Exceptions\LoginRequiredException();
+	}
+
+	if(!Session::$User->Benefits->CanReviewOwnArtwork){
+		throw new Exceptions\InvalidPermissionsException();
+	}
+
+	session_start();
+
+	$exception = HttpInput::SessionObject('exception', Exceptions\AppException::class);
+
+	$artist = Artist::GetByUrlName(HttpInput::Str(GET, 'artist-url-name') ?? '');
```

Comment:
I don't think we need `??` here because passing null will still throw an exception, but will also avoid a DB call

Reply:
Done